README.md

```diff
@@ -27,7 +28,15 @@ python -m venv venv
 source venv/bin/activate
 pip install --upgrade pip
 pip install -r requirements.txt
-
+```
+* Install kubectl.
```

Comment:
Maybe this should be under Prerequisites chapter.

Reply:
true. I will move it there

Reply:
Good catch @kivaisan